src/components/InlineCodeBlock/WrappedText.tsx

```diff
@@ -61,7 +62,21 @@ function WrappedText({children, wordStyles, textStyles}: WrappedTextProps) {
                     style={styles.codeWordWrapper}
                 >
                     <View style={[wordStyles, colIndex === 0 && styles.codeFirstWordStyle, colIndex === rowText.length - 1 && styles.codeLastWordStyle]}>
-                        <Text style={[textStyles, !containsEmoji(colText) && styles.codePlainTextStyle]}>{colText}</Text>
+                        <Text style={[textStyles, !containsEmoji(colText) && styles.codePlainTextStyle]}>
+                            {Array.from(colText).map((char, charIndex) =>
```

Comment:
Can you elaborate why do we need this change?

Reply:
@eh2077 per native, we continue utilizing [getCurrentData](https://github.com/Expensify/App/blob/main/src/components/InlineCodeBlock/getCurrentData.ts). WrappedText has intricate logic that solely supports string input. The modification above aims to wrap emojis with the Inner Text component for the ability to customize its style using `emojiDefaultStyles`.